### PR TITLE
1478: es7 update

### PR DESCRIPTION
**Pull request by flacoman91**
- fixes the pager counts
- reset pager & requery when the following changes: 
  - sort
  - size
  - Search Text
  - Search Field
- Added Cypress testing for List view, fixed broken cypress tests

### CODE REVIEW

File: src/actions/complaints.jsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ export function getAggregations() {
 export function getComplaints() {
   return ( dispatch, getState ) => {
     const store = getState()
-    const qs = store.query.queryString + '&no_aggs=true'
```
Comment (higs4281) [inline src/actions/complaints.jsx:108]: Do we need aggregations in the complaint request, or was this just to get the request cycle working? 
Not running aggregations on every request is a bit more efficient. Not huge.

**Comment by flacoman91** [comment]
I removed it to get it running.

**Comment by higs4281** [comment]
Thanks. 

We also need aggregations to find the total search count when there are 10K or more hits, so it makes sense to have them included.

File: setup.py [hunk 2]
```diff
@@ -39,7 +39,7 @@ def read_file(filename):
 
 setup(
     name='ccdb5_ui',
-    version=get_git_version(),
+    version='2.3.3-dev5',
```
Comment (higs4281) [inline setup.py:42]: Not related to your PR, but could we revert? I made this change while struggling to get ccdb deployed to DEV5, and we can switch back now.

```suggestion
    version=get_git_version(),
```

**Review by higs4281 — APPROVED** [review]
It's looking good from here.

**Pull request merged**